src/Table.luna

```diff
@@ -769,18 +1122,87 @@ class Table:
         quart3 = col.quantile 0.75
         Table.fromColumns [mean, std, min, quart1, median, quart3, max]
 
+    # Returns table with unique values counts for selected column.
+    #
+    # > import Dataframes.Column
+    # > import Dataframes.Types
+    # > import Dataframes.Table
+    # >
+    # > def main:
+    # >     l1 = [1,2,3,4,5]
+    # >     l2 = [11,12,13,11,15]
+    # >     col1 = Column.fromList "col1" Int64Type l1
+    # >     col2 = Column.fromList "col2" Int64Type l2
+    # >     table = Table.fromColumns [col1 , col2]
+    # >     unique = table.countValues "col2"
+    # >     None
+    #
+    # `columnName`: Text value, name of the column to summary unique values for.
+    #
+    # `return`: Table with all unique values in first column and their counts in second for selected column.
+
     def countValues columnName:
         self.at columnName . toColumn . countValues
 
     # aggregateBy :: Text -> [(Text, [AggregateFunction])] -> Table
+    # Aggregate using one or more operations over specified columns. 
+    #
+    # Predefined aggregate functions are available: Minimum, Maximum, Mean, Length, Median, First, Last, Sum, RSI, Std.
+    #
+    # > import Dataframes.Column
+    # > import Dataframes.Types
+    # > import Dataframes.Table
+    # >
+    # > def main:
+    # >     l1 = [1,2,1,4,2]
+    # >     l2 = [11,12,13,14,15]
+    # >     l3 = [21,22,23,24,25]
+    # >     l4 = [31,32,33,34,35]
+    # >     col1 = Column.fromList "col1" Int64Type l1
+    # >     col2 = Column.fromList "col2" Int64Type l2
+    # >     col3 = Column.fromList "col3" Int64Type l3
+    # >     col4 = Column.fromList "col4" Int64Type l4
+    # >     table = Table.fromColumns [col1 , col2, col3, col4]
+    # >     aggregated = table.aggregateBy "col1" [("col2", [Mean]),("col4", [Minimum, Median])]
+    # >     None
+    # 
+    # `keyColumnName`: Column to aggregate over.
+    # `aggregations`: List of aggregations. Single aggregation is a two argument tuple. First element is a name of a column for aggregation and second is a list of aggregate functions.
+    #
+    # `return`: `Table` value containing given aggregations calculated over key column for columns and aggreate functions from the second argument.
```

Comment:
aggreate -> aggregate

Reply:
Not sure if this wording is clear what this really does. That sounds almost as if aggregations are applied over key column, not the columns to aggregate over.

Consider:
`Table` value where each column is a result of applying an aggregation function over a column (as specified by the list).